docs/source/conf.py

```diff
@@ -213,10 +214,18 @@
     # Generate docs even if an item has no docstring.
     "undoc-members": True,
     # Don't document things with a leading underscore.
-    "private-members": False,
+    "private-members": True,
+    # Include special members.
+    "special-members": "__init__",
+    # Show class inheritance.
+    "show-inheritance": True,
 }
 # Display type hints in the description instead of the signature.
 autodoc_typehints = "description"
+# Mock problematic imports
+autodoc_mock_imports = [
+    "networkx",
```

Comment:
I was getting a handful of class reference target errors that I think are just due to networkx not followin the very strict rules we need to following based on our autodoc implementation. I'm not totally clear on what exactly we're _losing_ by doing this, but a quick scan through a few of the docs look ok.